### PR TITLE
New package: glab-1.14.0

### PR DESCRIPTION
**Pull request by Gottox**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by Piraty** [comment]
`srcpkgs/gitlab-runner` ?

File: srcpkgs/glab/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'glab'
+pkgname=glab
+version=1.14.0
+revision=1
+build_style=go
+go_import_path=github.com/profclems/glab
+go_package="${go_import_path}/cmd/glab"
+short_desc="Official GitLab CI runner written in Go"
```
Comment (Piraty) [inline srcpkgs/glab/template:8]: may be you copied another template and didn't modify relevant parts?

**Review by Piraty — CHANGES_REQUESTED** [review]

File: srcpkgs/glab/files/gitlab-runner/run [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec gitlab-runner run
```
Comment (Piraty) [inline srcpkgs/glab/files/gitlab-runner/run:2]: leftover

File: srcpkgs/glab/update [hunk 3]
```diff
@@ -0,0 +1,3 @@
+site="https://gitlab.com/gitlab-org/gitlab-runner/tags"
```
Comment (Piraty) [inline srcpkgs/glab/update:1]: leftover

**Review by Piraty — CHANGES_REQUESTED** [review]

**Comment by Gottox** [comment]
thx for the pointers! :)

File: srcpkgs/glab/template [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+version=1.14.0
+revision=1
+build_style=go
+go_import_path=github.com/profclems/glab
+go_package="${go_import_path}/cmd/glab"
+short_desc="Command line tool bringing GitLab's features to your command line"
+maintainer="Enno Boland <gottox@voidlinux.org>"
+license="MIT"
+homepage="https://github.com/profclems/glab"
+distfiles=https://github.com/profclems/$pkgname/archive/v$version.tar.gz
```
Comment (ericonr) [inline srcpkgs/glab/template:12]:
```suggestion
distfiles="https://github.com/profclems/glab/archive/v$version.tar.gz"
```

File: srcpkgs/glab/template [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+checksum=927206802cda67f0b725d0ed2b355a67ab9eb560a9e8b8c6e4ca994d9aba3daa
+
+post_install() {
+	vlicense ${wrksrc}/LICENSE
```
Comment (ericonr) [inline srcpkgs/glab/template:16]: I think this should be just `vlicense LICENSE`

**Review by Piraty — APPROVED** [review]

**Pull request merged**